data/org_cluster_map_handler.py

```diff
@@ -0,0 +1,132 @@
+    :return: None
+    """
+
+    # Move leaflet_dist folder to static folder
+    leaflet_source_path = f'{os.getcwd()}/{output_dir}/leaflet_dist/'
+    leaflet_destination_path = f'{os.getcwd()}/static/leaflet_dist/'
+
+    # Remove existing leaflet_dir if exists
+    for root, dirs, files in os.walk(leaflet_destination_path):
+        for file in files:
+            os.remove(os.path.join(leaflet_destination_path, file))
+        os.rmdir(root)
+
+    os.renames(leaflet_source_path, leaflet_destination_path)
+
+    # Move org_locations.js to static folder
+    locations_source_path = f'{os.getcwd()}/{output_dir}/org-locations.js'
+    locations_destination_path = f'{os.getcwd()}/static/org-locations.js'
+    os.rename(locations_source_path, locations_destination_path)
+
+    # Make changes in map.html to support django syntax with needed CSS
+    with open('{}/map.html'.format(output_dir)) as f:
+        django_supported_htmls = []
+        for html_line in f:
+            line = html_line.strip()
+            if line.__contains__('<html>'):
+                django_supported_htmls.append('{% load staticfiles %}\n')
+                django_supported_htmls.append(line + '\n')
+            elif line.__contains__('</head>'):
+                adjust_prop = '''
+                <style>
+                    #map {
+                    width: 60%;
+                    height: 300px;
+                    margin: auto;
+                    box-shadow: 0px 0px 25px 2px;
+                    }
+                    @media only screen and (max-width:750px){
+                        #map {
+                            width: 90%
+                        }
+                    }
+                </style>\n
+                '''
+                meta_charset = '<meta charset="utf-8">'
+                adjust_prop = adjust_prop.strip().replace('    ', '')
+                django_supported_htmls.insert(6, meta_charset + '\n')
+                django_supported_htmls.append(adjust_prop + '\n')
+                django_supported_htmls.append(line + '\n')
+            elif line.__contains__('https://'):
+                line = line.replace('https:', '').replace(' />', '>')
+                django_supported_htmls.append(line.strip() + '\n')
+            elif line.__contains__('<link '):
+                line = line.replace('href="', 'href="{% static \'')
+                line = line.replace('.css', '.css\' %}').replace(' />', '>')
+                django_supported_htmls.append(line + '\n')
+            elif line.__contains__('<script '):
+                line = line.replace('src="', 'src="{% static \'')
+                line = line.replace('.js', '.js\' %}')
+                if line.__contains__(' type="text/javascript"'):
+                    line = line.replace(' type="text/javascript"', '')
+                django_supported_htmls.append(line + '\n')
+            elif line.__contains__('Mouse over') or len(line) == 0:
+                pass
```

Comment:
What is the point of having this `elif` statement if you're not doing anything in it?

Reply:
Want to ignore unnecessary blank lines and the text. Will add a `not` to this and add the below code to this. It will execute in dame manner 👍🏻